spec/services/process_submission_service_spec.rb

```diff
@@ -189,7 +189,7 @@
         stub_request(:get, 'http://fake_service_token_cache_root_url/service/service-slug').to_return(status: 200, body: { token: service_slug_secret }.to_json)
       end
 
-      it 'dispatches 1 email for each submission email attachment' do
+      xit 'dispatches 1 email for each submission email attachment' do
```

Comment:
It now sends 5 instead of 4 emails, which is expected in this transition period as we test.
Disabling the old PDF functionality will bring this back down to 4.

Reply:
Cool done